CHANGELOG.MD

```diff
@@ -1,30 +1,20 @@
-# Change Log
-
-All notable changes to this project will be documented in this file. This project adheres to [Semantic Versioning](https://semver.org/).
-
-## v1.1.0 - 2023-11-21
-
-> **Feature**: Started logging source url path with feedback submission. Viewable in dashboard. [🎟️DESENG-415](https://apps.itsm.gov.bc.ca/jira/browse/DESENG-415)
-
-> **Bug Fix**: Removed a duplicate service class. [🎟️DESENG-429](https://apps.itsm.gov.bc.ca/jira/browse/DESENG-429)
-
-
-## v1.0.1 - 2023-10-26
-
-> **Bug Fix**: Upgraded BC-Sans font to newest version. [🎟️DESENG-413](https://apps.itsm.gov.bc.ca/jira/browse/DESENG-413)
-
-> **Bug Fix**: Engagements will now open in the same browser window/tab, not a new one. [🎟️DESENG-421](https://apps.itsm.gov.bc.ca/jira/browse/DESENG-421)
-
-> **Bug Fix**: Update sample .env files - [🎟️DESENG-414](https://apps.itsm.gov.bc.ca/jira/browse/DESENG-414)
->- Sample .env files have been updated to reflect the current state of the project.
->- *Breaking*: Keycloak URLs and resources now point to the BC Government's SSO service when using `sample.env` as a baseline
->- *Breaking*: The `met_api` module has been updated slightly to consume Pathfinder SSO's API schema.
-
+## November 29, 2023
```

Comment:
I assume you'd like to stick with this changelog format going forward?

Reply:
That was my thought. What do you think?